include/dxc/Support/dxcapi.use.h

```diff
@@ -31,7 +31,7 @@ class DllLoader {
 
 public:
   DllLoader() = default;
-  DllLoader(const DllLoader &) = delete;
+  DllLoader(const DllLoader &) = default; // needed for HlslIntellisenseSupport
```

Comment:
Please mark this as deleted and remove the HlslIntellisenseSupport move constructor.

Reply:
Do we want to delete this solely because it's an interface class? Or is there some other reason?

Reply:
There's no way for someone who has a pointer to a `DllLoader` instance to safely copy or move it, so we shouldn't have copy/move constructors for it.  Plus, since there's no actual need to ever copy these we shouldn't be spending time trying to make it work.  As it is now, the default copy constructor for `SpecificDllLoader` will end up leaking the dll HANDLE, so we don't want people trying to copy that.